www/source/docs/habitat-cli.html.md

```diff
@@ -1,7 +1,7 @@
 <!-- This is a generated file, do not edit it directly. See https://github.com/habitat-sh/habitat/blob/master/www/scripts/generate-cli-docs.js -->
-
----
-title: hab CLI Reference
+ ---
```

Comment:
The header still doesn't generate correctly.
![Screen Shot 2020-01-29 at 10 26 05 AM](https://user-images.githubusercontent.com/4400151/73385377-eba69a00-4281-11ea-8b44-22cce1fe0446.png)

Reply:
Can we generate the comment *after* the frontmatter?

Reply:
This might be really simple. There's a blank space in front of the first dash. 

https://github.com/habitat-sh/habitat/blob/master/www/scripts/generate-cli-docs.js#L78

Reply:
Nice find @kagarmoe ... I've fixed this in https://github.com/habitat-sh/habitat/pull/7396/commits/25f17540ea00b8e4ee0d1be5e24ca92e1455c45e. Let me know what you think! :bowing_man:

Reply:
Re-fixed in https://github.com/habitat-sh/habitat/pull/7396/commits/334ff10f0e55e65473129355566a58af8fd554ee